iOS Client/Store/MBPodcastsStore.swift

```diff
@@ -52,30 +52,9 @@ class MBPodcastsStore {
                     podcasts.append(contentsOf: displayCasts)
                 }
             }
+            MBStore.sharedStore.dispatch(SetPodcastStreams(streams: [.pz, .mockingPulpit, .mockingCast]))
```

Comment:
I think it'd be fine to hard-code this into the initial state; if you still want to dispatch this action tho, I think u can reuse streams defined on line 30 of this file instead of making new array